subsys/nrf_security/configs/psa_crypto_config.h.template

```diff
@@ -221,7 +221,9 @@
 
 /* TF-M */
 #cmakedefine MBEDTLS_PSA_CRYPTO_SPM
+#ifndef MBEDTLS_PSA_CRYPTO_KEY_ID_ENCODES_OWNER
```

Comment:
I'm somewhat supsicious about this. Is this defined to something elsewhere?

Reply:
discussed offline